test/language/expressions/arrow-function/arrow/bind_2.js

```diff
@@ -0,0 +1,21 @@
+// Copyright 2015 Microsoft Corporation. All rights reserved.
+// This code is governed by the license found in the LICENSE file.
+
+/*---
+description: invoke arrow function using bind
+es6id: 14.2
+---*/
+
+var b = {
+    method: () => this
+ }
+
+var a = b.method;
+
+assert.sameValue(a.bind(null)(), this, "Invocation of arrow function failed using fn.bind(null).");
+
+assert.sameValue(a.bind(undefined)(), this, "Invocation of arrow function failed using fn.bind(undefined).");
+
+assert.sameValue(a.bind(this)(), this, "Invocation of arrow function failed using fn.bind(this).");
+
+assert.sameValue(a.bind({})(), this, "Invocation of arrow function failed using fn.bind({}).");

```

Comment:
Duplicates

- https://github.com/tc39/test262/blob/master/test/language/expressions/arrow-function/lexical-this.js